Extensions/xkit_preferences.css

```diff
@@ -1250,164 +1100,84 @@
 }
 
-.xkit-window-title {
-	color: black;
-	font-size: 19px;
-	padding-top: 18px;
-}
-
-.xkit-extension-setting.xkit-experimental-option {
-	/*background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAcAAAA8CAIAAAAljTfqAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAyJpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvIiB4bWxuczp4bXBNTT0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wL21tLyIgeG1sbnM6c3RSZWY9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9zVHlwZS9SZXNvdXJjZVJlZiMiIHhtcDpDcmVhdG9yVG9vbD0iQWRvYmUgUGhvdG9zaG9wIENTNSBNYWNpbnRvc2giIHhtcE1NOkluc3RhbmNlSUQ9InhtcC5paWQ6OTYwODRBMDAzNzM4MTFFM0E0MTFDNDE2OTFDMzA4QzciIHhtcE1NOkRvY3VtZW50SUQ9InhtcC5kaWQ6OTYwODRBMDEzNzM4MTFFM0E0MTFDNDE2OTFDMzA4QzciPiA8eG1wTU06RGVyaXZlZEZyb20gc3RSZWY6aW5zdGFuY2VJRD0ieG1wLmlpZDo5NjA4NDlGRTM3MzgxMUUzQTQxMUM0MTY5MUMzMDhDNyIgc3RSZWY6ZG9jdW1lbnRJRD0ieG1wLmRpZDo5NjA4NDlGRjM3MzgxMUUzQTQxMUM0MTY5MUMzMDhDNyIvPiA8L3JkZjpEZXNjcmlwdGlvbj4gPC9yZGY6UkRGPiA8L3g6eG1wbWV0YT4gPD94cGFja2V0IGVuZD0iciI/Ps5DRcUAAABySURBVHjaYvz8YD+PvMP3F+ff31zF8P8vn6IXkMuc4y/GyMjKLWPJwiH84+31n+9vAbksDIzMn+5vY2BgACphYGR8f2MlkMskqB4Gkfjy8ACnuIGgRjiQy/j//380Q7+/vMA4atuobaO2jdo25GwDCDAA+two9+RM17sAAAAASUVORK5CYII=);
-	background-repeat: no-repeat;
-	background-position: -3px 0px;*/
-	background: white url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAMAAAADCAYAAABWKLW/AAAAGklEQVQIW2NkYGCQBOLnQMzACCJgAjAOWAAALlYCHYeSYiEAAAAASUVORK5CYII=');
-
+.xkit-extension-setting.xkit-experimental-option:before, 
+.xkit-extension-setting.xkit-experimental-option, 
+.xkit-extension-setting.xkit-experimental-option:after {
+	background: repeat url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 1 7'%3E%3Cdefs%3E%3ClinearGradient id='exp' spreadMethod='repeat' x1='0' y1='0' x2='-1%' y2='10%'%3E%3Cstop offset='0' stop-color='transparent'/%3E%3Cstop offset='.5' stop-color='transparent'/%3E%3Cstop offset='.5' stop-color='%23ffd70038'/%3E%3C/linearGradient%3E%3C/defs%3E%3Crect width='1' height='7' fill='url(%23exp)'/%3E%3C/svg%3E") !important;
 }
 
-.xkit-button.disabled {
-	box-shadow: none;
-	opacity: 0.75;
-	cursor: default;
-	background: rgb(240, 240, 240);
-	position: relative;
-	top: 0px;
-	color: rgb(170,170,170) !important;
-	border: 1px dashed rgb(160,160,160) !important;
+.xkit-extension-setting.xkit-experimental-option:before, 
+.xkit-extension-setting.xkit-experimental-option:after {
```

Comment:
`::before` and `::after` are pseudo-elements, not pseudo-selectors, and should be marked as such with `::` instead of `:`

(this applies to some other changes i can see; i won't mark them all individually)

[Pseudo-elements: Syntax - MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements#Syntax)